### ISSUE TITLE
Manually-entered seed is not applied to multiple images in batch

### ISSUE THREAD

**Issue by dngrlux** [divamgupta/diffusionbee-stable-diffusion-ui#215]
Use case: I wanted to create 14 slight variations of a seed rendered image so that I can experiment with lower guidance scales and photobash the best bits together. I manually entered a seed value.

Expected: 14 variations of the prompt using the same seed as entered.

Actual: The first image used the entered seed, and then continued to add 1234 to the seed for each additional image.

Workaround: Set the number of images to 1 and generate 14 individual images with that seed, babysitting the process.

The interface shows the label "Seed" whereas it is acting as a "Starting Seed."

**Comment by Leland** [comment]
Yeah, I agree. This should probably be the manual seed for each image (would also close #185). Having one manual seed image and then n-1 random seed images in a batch does not seem like the behavior a typical user would expect.
